dockers/Screenshotter/screenshotter.js

```diff
@@ -393,7 +440,8 @@ function takeScreenshot(key) {
              * output file name for one of these cases, we accept both.
              */
             key += "_alt";
-            file = path.join(dstDir, key + "-" + opts.browser + ".png");
+            basename = key + basename.slice(key.length - 4);
```

Comment:
We turn `"Lap-firefox-nq"` into `"Lap_alt-firefox-nq"`. We know that the string starts with `"Lap-"` at this point, and that's what we remove. Would you prefer I change this to `basename = basename.replace("Lap", "Lap_alt")` for better maintainability?